snmp/tests/test_e2e_core_vs_python.py

```diff
@@ -484,7 +484,7 @@ def test_e2e_profile_isilon(dd_agent_check):
 
 def test_e2e_profile_meraki_cloud_controller(dd_agent_check):
     config = common.generate_container_profile_config('meraki-cloud-controller')
-    assert_python_vs_core(dd_agent_check, config)
+    assert_python_vs_core(dd_agent_check, config, tags_to_skip=['mac_address'])
```

Comment:
Note for reviewers: We need to skip `mac_address` tag for python vs core tests since their format differ.